app.js

```diff
@@ -27,6 +28,12 @@ var steamUserMapping = {
     'boomsy': 14046169
 };
 
+var dotaHeroList = JSON.parse(fs.readFileSync('heroes.json', 'utf8'));
+var dotaHeroMapping = {};
+for (var i = 0; i < dotaHeroList.length; i++) {
+    dotaHeroMapping[dotaHeroList[i].id] = dotaHeroList[i].localized_name;
```

Comment:
Would a `map` function work better for this?

Reply:
Not sure if you mean

  1) we should use an [ES6 Map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map) instead of the plain JS object in using for `dotaHeroMapping`
  2) `dotaHeroMapping` should actually be a function whose parameter is the `hero_id` and which returns the hero name
  3) `dotaHeroMapping` should be created/filled in by using `Array.prototype.map()` (I actually think we would need `reduce()` here because we reduce an array, `dotaHeroList`, to a single object)